run_perf_tests.py

```diff
@@ -307,25 +301,56 @@ def analyze_data_files(dest_dir):
         return 1
 
 
-def get_testcase_name(dest_dir, full_filename):
-    dump_results = parse_dump_file(full_filename)
-    testcase_name = dump_results['Testname']
-    testcase_names.append(testcase_name)
-
-
-def xml(dest_dir, jank_perc, execution_time):
+def xml(dest_dir,test_data_dir):
     xml_file_dir = os.path.join(dest_dir, 'app/build/outputs/androidTest-results/connected')
     for file in os.listdir(xml_file_dir):
         xml_file_name = file
     tree = ElementTree.ElementTree(file = xml_file_dir + '/' + xml_file_name)
+
     for element in tree.findall('testcase'):
         name = element.get('name')
-        for testcase_name, jank_perc, execution_time in zip(testcase_names,list_for_jank_perc,list_for_execution_time):
-            if name == testcase_name:
-                element.set('jank-percentage', jank_perc)
-                element.set('execution-time', execution_time)
+        classname = element.get('classname')
+        folder_name = classname + '_' + name
+        parsing_data_dir = os.path.join(test_data_dir + '/testdata/testdata/', folder_name)
+        for dir_name, sub_dir_list, file_list in os.walk(parsing_data_dir):
```

Comment:
Here we're analysing test data for a certain test case, and I think `os.listdir()` more suitable for this case.